packages/workflow/settings/index.js

```diff
@@ -49,7 +49,7 @@ const settings = {
 
   include() {
     // Allow developers to add their own node_modules include path
-    const userInclude = this.configuration.development.babelInclude;
+    const userInclude = this.configuration.development.babelInclude || [];
```

Comment:
The Joi schema should be enforcing that `babelInclude` will be an array and default to `[]` if none is provided, right? https://github.com/Availity/availity-workflow/blob/4d2cac080875c9d55f72df38af3b3e09f71ea3d2/packages/workflow/settings/schema.js#L57-L61